sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/SmsChannelProperties.cs

```diff
@@ -34,14 +34,14 @@ public SmsChannelProperties()
         /// <param name="accountSID">The Sms account SID. Value only returned
         /// through POST to the action Channel List API, otherwise
         /// empty.</param>
+        /// <param name="isEnabled">Whether this channel is enabled for the
+        /// bot</param>
         /// <param name="authToken">The Sms auth token. Value only returned
         /// through POST to the action Channel List API, otherwise
         /// empty.</param>
-        /// <param name="isEnabled">Whether this channel is enabled for the
-        /// bot</param>
         /// <param name="isValidated">Whether this channel is validated for the
         /// bot</param>
-        public SmsChannelProperties(string phone, string accountSID, string authToken, bool isEnabled, bool? isValidated = default(bool?))
+        public SmsChannelProperties(string phone, string accountSID, bool isEnabled, string authToken = default(string), bool? isValidated = default(bool?))
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
will add fallback constructor for back compat.